wuzz.go

```diff
@@ -547,6 +549,28 @@ func (a *App) SetKeys(g *gocui.Gui) {
 		a.closePopup(g, "method-list")
 		return nil
 	})
+
+	g.SetKeybinding("save-dialog", gocui.KeyEnter, gocui.ModNone, func(g *gocui.Gui, v *gocui.View) error {
+		defer a.closePopup(g, "save-dialog")
+
+		saveLocation := getViewValue(g, "save-dialog")
+
+		if len(a.history) == 0 {
+			return nil
+		}
+		req := a.history[a.historyIndex]
+		if req.RawResponseBody == nil {
+			return nil
+		}
+
+		err := ioutil.WriteFile(saveLocation, req.RawResponseBody, 0755)
```

Comment:
It would be useful to display an error message if something went wrong (e.g. permission problems).
Other note: `0755` is unnecessarily permissive, isn't `0644` enough?